units/src/amount.rs

```diff
@@ -1835,6 +1852,48 @@ mod tests {
     #[test]
     fn from_int_btc_panic() { Amount::from_int_btc(u64::MAX); }
 
+    #[test]
+    fn test_signed_amount_try_from_amount() {
+        let ua_positive = Amount::from_sat(123);
+        let sa_positive = SignedAmount::try_from(ua_positive).unwrap();
+        assert_eq!(sa_positive, SignedAmount(123));
+
+        let ua_zero = Amount::ZERO;
+        let sa_zero = SignedAmount::try_from(ua_zero).unwrap();
+        assert_eq!(sa_zero, SignedAmount(0));
+
+        let ua_max = Amount::MAX;
+        let sa_max = SignedAmount::try_from(ua_max);
```

Comment:
should probably just be `error` or `e` or something, right?  This isn't really SignedAmount type anymore.

Reply:
ok I will make all the test cases more clean and squash the commits.
Thanks for the review